app/views/content/landing/what-teaching-taught-me/_collage.html.erb

```diff
@@ -0,0 +1,25 @@
+<div class="row">
+  <div class="grey col">
+    <section class="col col-720 col-space-0 col-center associated-logos">
+      <div class="inset">
+        <p><strong>
+        In partnership with
+        </strong></p>
+      </div>
+      <div class="associated-logos-holder">
+        <%= image_pack_tag("static/images/content/grow/buzzfeed-red-logo-text.png", alt: "Buzzfeed logo") %>
```

Comment:
This looks a bit low-res and also squashed horizontally @garethwilliamsdesign ?

I also think we should check with the partnerships team whether we need the DfE logo - I suspect we don't (it wasn't in their draft doc) which might make things easier?

Reply:
Yes, as I mentioned on Slack last night, the BuzzFeed logo looked great on my machine, but went all squiffy on GitHub. I'll fix this issue once it's been decided how many logos we want.